__tests__/app.test.js

```diff
@@ -103,4 +95,43 @@ afterAll(() => db.end());
                 })
             })
         })
+    })
```

Comment:
As you say in your PR message, need to update `endpoints.json`

Reply:
resolved